src/views/create-account-view.ts

```diff
@@ -3,11 +3,44 @@ import '../ga-heading';
 import '@material/mwc-textfield';
 import '@material/mwc-button';
 
-import { css, customElement, html, LitElement, TemplateResult } from 'lit-element';
+import { css, customElement, html, LitElement, property, TemplateResult } from 'lit-element';
 
 @customElement('create-account-view')
 export class CreateAccountView extends LitElement
 {
+    @property()
+    userName = '';
+    userMail = '';
+    userPass = '';
+    userSecPass = '';
+
+
+
+    private async createAccount(): Promise<void>
+    {
+        console.log(this.userName);
```

Comment:
No console logs in branches that should be merged. At best also not in any commits.